server/cpho/views/indicator_metadata.py

```diff
@@ -157,7 +157,14 @@ def clean_year(self):
         year = self.cleaned_data["year"]
 
         if year is None or year == "":
-            return None
+            if self.cleaned_data["oecd_country"].name_en in ["OECD"]:
+                return None
```

Comment:
Why don't OECD rows have years?

Reply:
I asked HSO about this, they said some OECD rows don't have years
![image](https://github.com/PHACDataHub/cpho-phase2/assets/39681045/569566d8-cd22-4a69-9b63-920eced65a7c)